typescript/example_code/README.rst

```diff
@@ -0,0 +1,14 @@
+###############################
+TypeScript AWS SDK examples
```

Comment:
"TypeScript AWS SDK examples" -> "TypeScript code examples" -- i.e. the CDK is not, strictly speaking, an "SDK".